src/editor.ts

```diff
@@ -0,0 +1,132 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  css,
+  CSSResult,
+} from 'lit-element';
+import '@polymer/paper-input/paper-input';
+import '@polymer/paper-toggle-button/paper-toggle-button';
+import { superstruct } from 'superstruct';
+
+import { BoilerplateConfig } from './index';
```

Comment:
this is a circular dependency. maybe we can move BoilerplateConfig to its own file?